propolis/Cargo.toml

```diff
@@ -21,7 +21,7 @@ viona_api = { path = "../viona-api" }
 usdt = { version = "0.2.1", default-features = false }
 tokio = { version = "1", features = ["full"] }
 futures = "0.3"
-crucible = { git = "https://github.com/oxidecomputer/crucible", branch = "main", optional = true }
```

Comment:
This change is here just to allow things to compile.

When this (and the crucible side) has been approved, this change can come out and main will become the branch to use for crucible.

Reply:
Please include a short comment in the toml itself to that effect?